org-code-javabuilder/lib/src/test/java/org/code/neighborhood/GridTest.java

```diff
@@ -0,0 +1,60 @@
+package org.code.neighborhood;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+public class GridTest {
+
+  @Test
+  void constructorBuildsGrid() {
+    GridSquare s = new GridSquare(1);
+    GridSquare[][] squares = new GridSquare[1][1];
+    squares[0][0] = s;
+    Grid grid = new Grid(squares);
+    assertTrue(grid.validLocation(0, 0));
+    assertFalse(grid.validLocation(0, 1));
+    assertFalse(grid.validLocation(1, 0));
```

Comment:
NIT: Looks like this test gets duplicated by the following test. If that's so, leave it out so there are fewer places to update when changing code in the future..